lib/school_house_web/views/html_helpers.ex

```diff
@@ -35,12 +35,31 @@ defmodule SchoolHouseWeb.HtmlHelpers do
         {"#", "cursor-not-allowed"}
       end
 
-    link(contents,
-      class: "#{class} #{additional_classes}",
-      to: destination
+    additional_classes = if Lessons.coming_soon?(name), do: additional_classes <> " mr-2", else: additional_classes
+
+    content_tag(
+      :div,
+      [
+        link(contents,
+          class: "#{class} #{additional_classes}",
+          to: destination
+        ),
+        Lessons.coming_soon?(name) |> maybe_coming_soon_badge()
```

Comment:
```suggestion
        name |> Lessons.coming_soon?() |> maybe_coming_soon_badge()
```

We try to follow this style guide: https://github.com/christopheradams/elixir_style_guide#avoid-single-pipelines